src/config/dynamic-theme-fixes.config

```diff
@@ -14248,6 +14248,18 @@ INVERT
 
 ================================
 
+wealthfront.com
+
+INVERT
+.dashboard-path-goal-icon-component
+
+CSS
+.default__9vSXGQvh8MlK1JxJBq-k4 {
```

Comment:
Can this CSS selector be changed? As such CSS Selectors that include randomness will most likely change every few so.

Reply:
> I think you're right that it will change, but it is still the same after almost 7 months.

I prefer to not set such code-style, as other new contributes will point at this and say "but it's allowed here". So I'd prefer a consistent way of not using such random CSS selector.